ui/app/AppLayouts/Wallet/services/dapps/types/SessionRequestResolved.qml

```diff
@@ -48,30 +46,11 @@ QObject {
     /// maps to Constants.TransactionEstimatedTime values
     property int estimatedTimeCategory: 0
 
-    function resolveDappInfoFromSession(session) {
```

Comment:
This is not needed. The dapp metadata is known in advanced. This is now provided by the `DAppsModel` and we don't need to always do asynchronous calls to the sdk to get this kind of info.